apps/src/aichat/views/UserChatMessageEditor.tsx

```diff
@@ -40,6 +55,25 @@ const UserChatMessageEditor: React.FunctionComponent<{
     }
   }, [disabled]);
 
+  const isChatGpt = useAppSelector(
+    state =>
+      state.aichat.currentAiCustomizations.selectedModelId ===
+      AiChatModelIds.CHATGPT
+  );
+  const multimodalEnabled = isChatGpt && queryParams('multimodal') === 'true';
```

Comment:
Not sure what the standard is these days with `enableExperiments`/session storage vs straight up query params (FWIW I like the simpler query param), but just noting there's a helper called `isEnabledAllowingQueryString` that allows both :)

Reply:
Oh yeah good question, I'm not sure either 🙂 I figured with a query param, users have to be more deliberate which is maybe better since this is a somewhat unfinished feature?